includes/gateways/paypal/functions.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * PayPal Commerce Functions
+ *
+ * @package    easy-digital-downloads
+ * @subpackage Gateways\PayPal
+ * @copyright  Copyright (c) 2021, Sandhills Development, LLC
+ * @license    GPL2+
+ * @since      2.11
+ */
+
+namespace EDD\PayPal;
+
+use EDD\PayPal\Exceptions\Authentication_Exception;
+
+/**
+ * Determines whether or not there's a valid REST API connection.
+ *
+ * @param string $mode Mode to check (`live` or `sandbox`).
+ *
+ * @since 2.11
+ * @return bool
+ */
+function has_rest_api_connection( $mode = '' ) {
+	try {
+		$api = new API( $mode );
+
+		return true;
+	} catch ( Authentication_Exception $e ) {
+		return false;
+	}
+}
+
+/**
+ * Determines whether or not PayPal Standard should be enabled.
+ * This returns true if the store owner previously had a PayPal Standard connection but has not yet
+ * connected to the new REST API implementation.
+ *
+ * If PayPal Standard is enabled, then PayPal payments run through the legacy API.
+ *
+ * @param string $mode
+ *
+ * @since 2.11
+ * @return bool
+ */
+function paypal_standard_enabled( $mode = API::MODE_LIVE ) {
```

Comment:
Noting here in case the issue is here, but on the site where I have connected successfully to PayPal Commerce, the PayPal Standard settings and gateway are still showing for me.

Reply:
Pushed up a potential fix, if you could please check again?